### PR TITLE
Add documentation on upgrading packages

### PR DESCRIPTION
**Pull request by jgrund**
This patch adds some documentation for updating dependencies.

We can add some more docs on adding new deps in a later patch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+
+This decouples our dependency code from our application code. We can also package our application using this process.
+
+## Updating dependencies
+
+There are a few steps needed to update a dependency.
+
+In the directory you want to update:
+
+1. Upload a `.tgz` file cooresponding to the version you want to change
```
Comment (brianjmurrell) [inline README.md]: and `git add` that .tgz and `git rm` the one it's replacing

File: README.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+2. Update the `.spec` file.
```
Comment (brianjmurrell) [inline README.md]: and at minimum:
- update the _Version:_ and _Release:_ tags
- add a %changelog entry

File: README.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
 # manager-for-lustre-dependencies
 
-Dependencies needed for Manager For Lustre not available elsewhere
-
 [![Build Status](https://travis-ci.org/intel-hpdd/manager-for-lustre-dependencies.svg?branch=master)](https://travis-ci.org/intel-hpdd/manager-for-lustre-dependencies)
```
Comment (brianjmurrell) [inline README.md]: Any idea why this badge is not working?

Comment (jgrund) [inline README.md]: It's targeting master, but nothing is building on master.

Comment (brianjmurrell) [inline README.md]: Ahh.  Right.  Obvious now that you say it.

I guess that badge is going to be useless on this project.  :-(

Comment (jgrund) [inline README.md]: It's still useful as a shortcut to see the travis runs, but yeah

**Pull request merged**